vignettes/snomedizer.Rmd

```diff
@@ -0,0 +1,373 @@
+---
+title: "snomedizer: R Interface to the SNOMED CT Terminology Server REST API"
+date: '`r format(Sys.Date(), "%d %B %Y")`'
+output: 
+  rmarkdown::html_vignette:
+    toc: true
+    always_allow_html: yes
+    df_print: tibble
+vignette: >
+  %\VignetteIndexEntry{snomedizer: R Interface to the SNOMED CT Terminology Server REST API}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(dplyr)
+library(snomedizer)
+```
+
+# Overview
+
+## What is SNOMED CT?
+
+SNOMED CT is an international ontology of clinical terms. You can think of it as an encyclopedia on all things that are talked about in medicine, for instance: anatomical structures, diagnoses, medications, tests,  surgical procedures, pathogens such as SARS-CoV-2 or *Candidida auris*, or even clinical findings, such as a high temperature.
+
+SNOMED CT is used mainly in the UK:
```

Comment:
I would perhaps phrase the use a bit stronger along the lines of "All NHS healthcare providers in England must now use SNOMED CT for capturing clinical terms within electronic patient record systems. Northern Ireland, Scotland and Wales also have programmes of work underway in relation to SNOMED CT" taken from https://digital.nhs.uk/services/terminology-and-classifications/snomed-ct -- mainly to say that SnomedCT is not going to go away and better to work with it now